include/tester/vtcp/VTCPSession.hpp

```diff
@@ -18,8 +18,11 @@ class VTCPSession : public TCPSession
     std::function<void()> startManualTest,
     std::function<bsoncxx::document::value()> getHistoryHeaders,
     std::function<bsoncxx::document::value(const std::string &)> getHistoryLog):
-        TCPSession(std::move(socket), id, onMessageReceived, onDisconnect, startAutoTest, 
-            startManualTest, getHistoryHeaders, getHistoryLog)
+        TCPSession(std::move(socket), id, onMessageReceived, onDisconnect),
```

Comment:
`std::move` on all of the function variables you pass by value.

Reply:
done